tests/ui/doc/doc_comment_double_space_linebreak.stderr

```diff
@@ -0,0 +1,76 @@
+error: doc comment uses two spaces for a hard line break
+  --> tests/ui/doc/doc_comment_double_space_linebreak.rs:7:43
+   |
+LL |   //! Should warn on double space linebreaks  
+   |  ___________________________________________^
+LL | | //! in file/module doc comment
+   | |____^
+   |
```

Comment:
Have you tried changing the error span, like suggested here?

https://github.com/rust-lang/rust-clippy/pull/12876#discussion_r1713615334

```suggestion
  --> tests/ui/doc/doc_comment_double_space_linebreak.rs:7:43
   |
LL |   //! Should warn on double space linebreaks  
   |                                             ^^
   |
   |
```

It should be something simple, like taking a span from `lo_span` to `lo_span + 2`